plugins/maven/src/main/java/org/wildfly/swarm/plugin/maven/StopMojo.java

```diff
@@ -68,30 +69,14 @@ public void executeSpecific() throws MojoExecutionException, MojoFailureExceptio
         if (tmpDir.toFile().exists()) {
             File[] filesTmp = tmpDir.toFile().listFiles();
             for (File tmpFile : filesTmp) {
-                Matcher matcher = tempFilePattern.matcher(tmpFile.getName().toString());
+                Matcher matcher = tempFilePattern.matcher(tmpFile.getName());
                 if (matcher.matches()) {
-                    deleteRecursively(tmpFile);
+                    TempFileManager.deleteRecursively(tmpFile);
                 }
             }
         }
     }
 
-    public boolean deleteRecursively(File f) {
```

Comment:
The Maven plugin has a dependency on the `bootstrap` module. So, we don't really need to duplicate this code.